modules/http2/h2_mplx.c

```diff
@@ -173,6 +176,63 @@ static void m_stream_cleanup(h2_mplx *m, h2_stream *stream)
     }
 }
 
+static h2_c2_transit *c2_transit_create(h2_mplx *m)
+{
+    apr_allocator_t *allocator;
+    apr_pool_t *ptrans;
+    h2_c2_transit *transit;
+    apr_status_t rv;
+
+    /* We create a pool with its own allocator to be used for
+     * processing a request. This is the only way to have the processing
+     * independent of its parent pool in the sense that it can work in
+     * another thread.
+     */
+
+    apr_allocator_create(&allocator);
```

Comment:
This could fail on OOM, `return NULL` or simply `ap_abort_on_oom()` in this case?
Since allocation failures on ptrans will abort() I wonder if allocating ptrans itself shouldn't abort() too, thus here and in case of `apr_pool_create_ex()` below.
This'd make checking `c2_transit_{create,get}()` for NULL is the caller(s) moot.